src/utils/flag-utils.ts

```diff
@@ -37,3 +37,9 @@ export const setMvpFeatureFlag = (setFlag: SetFeatureFlag): void => {
   }
   setFlag(MVP_FLAG, enabled);
 };
+
+export const PROD_FLAG = 'PRODUCTION';
+
+export const setProdFeatureFlag = (setFlag: SetFeatureFlag): void => {
```

Comment:
I thought we were going to use unleash feature flags.

Reply:
@rohitkrai03 Added feature flagging using unleash. For now, the prod flag is enabled so reviewers can test this PR locally. Once this PR is merged we can disable the prod flag